src/assets/labContent.js

```diff
@@ -101,20 +105,19 @@ export default [
       subheader: "How do algorithms respond to stereotypical trends within data?",
       bodyText: [
         {
-          body: <text>The difference in ads shown across demographics is caused by the algorithm being trained on a <b>skewed sample</b> that reflects our society's biases. Much of the <b>correlations</b> it notices between race/gender and jobs have little to no actual <b>causal relationsip</b>, i.e. the kind of job a person looks for does not depend on their race or gender. However, a machine learning algorithm cannot tell when a trend it notices is correlation or causation, so it incorporates every trend it sees into its calculations for showing its users the most "optimal" ads.</text>
+          body: <text>The difference in ads shown across demographics is caused by the algorithm being trained on a <mark style={{backgroundColor: '#ffbdbd', borderRadius: '7px', padding: '0 5px 5px 5px'}}>skewed sample</mark> that reflects our society's biases. Much of the <i>correlations</i> it notices between race/gender and jobs have little to no actual <i>causal relationsip</i>, i.e. the kind of job a person looks for does not depend on their race or gender. However, a machine learning algorithm cannot tell when a trend it notices is correlation or causation, so it incorporates every trend it sees into its calculations for showing its users the most "optimal" ads.</text>
```

Comment:
Ok I'm going to stop with the inlined style comments, but I think that making them CSS classes would help reduce duplicate code!